src/syscalls/deprecated_business_logic_syscall_handler.rs

```diff
@@ -50,23 +50,34 @@ use num_traits::{One, ToPrimitive, Zero};
 /// Deprecated version of BusinessLogicSyscallHandler.
 #[derive(Debug)]
 pub struct DeprecatedBLSyscallHandler<'a, S: StateReader> {
+    /// Context of the transaction being executed
     pub(crate) tx_execution_context: TransactionExecutionContext,
     /// Events emitted by the current contract call.
     pub(crate) events: Vec<OrderedEvent>,
     /// A list of dynamically allocated segments that are expected to be read-only.
     pub(crate) read_only_segments: Vec<(Relocatable, MaybeRelocatable)>,
+    /// Manages execution resources
     pub(crate) resources_manager: ExecutionResourcesManager,
+    /// Address of the contract
     pub(crate) contract_address: Address,
+    /// Address of the caller
     pub(crate) caller_address: Address,
+    /// Messages from L2 to L1
     pub(crate) l2_to_l1_messages: Vec<OrderedL2ToL1Message>,
+    /// Context information related to the current block
     pub(crate) block_context: BlockContext,
+    /// Pointer to transaction information
     pub(crate) tx_info_ptr: Option<MaybeRelocatable>,
+    /// State of the storage related to Starknet contract
     pub(crate) starknet_storage_state: ContractStorageState<'a, S>,
+    /// Information about internal calls
```

Comment:
This should be something like "List of internal calls during the syscall execution"